src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line = "________________________________________";
+        ArrayList<Task> taskList = new ArrayList<>();
```

Comment:
Perhaps you should name this as tasks, as per the coding standards of the course